package.json

```diff
@@ -1,160 +1,160 @@
-    "eslint-plugin-monorepo": "^0.3.2",
-    "eslint-plugin-prettier": "^5.1.3",
-    "eslint-plugin-promise": "^6.1.1",
-    "eslint-plugin-react": "^7.34.1",
-    "eslint-plugin-react-hooks": "^4.6.0",
-    "eslint-plugin-react-native": "^4.1.0",
-    "eslint-plugin-storybook": "^0.8.0",
-    "eslint-plugin-testing-library": "^6.2.0",
-    "fast-glob": "^3.2.11",
-    "jest": "^29.7.0",
-    "jest-environment-jsdom": "^29.7.0",
-    "jest-extended": "^4.0.0",
-    "jest-serializer-html": "^7.1.0",
-    "jest-specific-snapshot": "^5.0.0",
-    "less": "^4.2.0",
-    "less-loader": "^11.1.3",
-    "nyc": "^15.1.0",
-    "prettier": "^3",
-    "react": "16.14.0",
-    "react-dom": "16.14.0",
-    "react-json-view": "^1.19.1",
-    "react-popper": "^2.0.0",
-    "react-router": "5.2.1",
-    "react-router-dom": "5.3.0",
-    "react-transition-group": "^4.4.1",
-    "react-window": "^1.8.5",
-    "rollup": "^2.70.2",
-    "rollup-plugin-auto-external": "^2.0.0",
-    "rollup-plugin-executable-output": "^1.3.0",
-    "rollup-plugin-filesize": "^9.1.1",
-    "rollup-plugin-preserve-shebangs": "^0.2.0",
-    "rollup-plugin-styles": "^4.0.0",
-    "sloc": "^0.2.1",
-    "storybook": "^7.6.17",
-    "style-loader": "^3.3.3",
-    "typescript": "^5.4.2",
-    "typescript-coverage-report": "^0.7.0",
-    "vite-plugin-istanbul": "^5.0.0",
-    "vite-plugin-turbosnap": "^1.0.3",
-    "winston": "^3.7.2"
-  },
-  "resolutions": {
-    "@types/jest": "29",
-    "@types/react": "16",
-    "@types/react-dom": "16"
-  },
-  "scripts": {
-    "gen:parsers": "yarn --cwd packages/kas gen:parsers",
-    "prebuild": "yarn gen:parsers",
-    "build": "rollup -c config/build/rollup.config.js",
-    "build:types": "yarn tsc --build tsconfig-build.json",
-    "build:prodsizecheck": "yarn build --configEnvironment='production'",
-    "watch": "yarn build --watch",
-    "clean": "rm -rf packages/*/dist && rm -rf packages/*/node_modules && rm -rf .nyc_output && rm -f packages/*/*.tsbuildinfo",
-    "coverage": "cross-env NODE_OPTIONS=--max_old_space_size=8192 yarn run jest --coverage",
-    "coverage:types": "cross-env NODE_OPTIONS=--max_old_space_size=8192 typescript-coverage-report",
-    "dev": "yarn --cwd dev dev",
-    "lint": "eslint . --ext .js --ext .jsx --ext .ts --ext .tsx",
-    "lint:timing": "cross-env TIMING=1 yarn lint",
-    "publish:ci": "utils/pre-publish-check-ci.ts && git diff --stat --exit-code HEAD && yarn build && yarn build:types && changeset publish",
-    "sloc": "sloc packages --exclude node_modules",
-    "test": "yarn jest",
-    "storybook": "storybook dev -p 6006",
-    "start": "storybook dev -p 6006",
-    "build-storybook": "storybook build",
-    "cypress": "cypress open --component",
-    "cypress:ci": "cross-env BABEL_COVERAGE=1 cypress run --component --env CYPRESS_COVERAGE=1",
-    "format": "prettier --write .",
-    "typecheck": "tsc"
-  },
-  "dependencies": {
-    "tiny-invariant": "^1.3.1"
-  }
+    "name": "perseus",
```

Comment:
This file was using 2-spaces indentation, but our whole repo is set to 4-spaces.... so turn of "Showing whitespace" to view this PR).

<img width="268" alt="image" src="https://github.com/Khan/perseus/assets/77138/b944ecdf-9464-48fd-9235-5ed364d825b7">